monitor/app/staking/metrics.go

```diff
@@ -0,0 +1,36 @@
+package staking
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	rewardsAvg = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "monitor",
+		Subsystem: "staking",
+		Name:      "avg_rewards",
+		Help:      "Average staking rewards percentage [0,1]",
+	})
+
+	delegatorsCount = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "monitor",
+		Subsystem: "staking",
+		Name:      "delegators",
+		Help:      "Number of unique delegators",
+	})
+
+	stakeAvg = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "monitor",
+		Subsystem: "staking",
+		Name:      "avg_stake",
```

Comment:
best add `_gwei` to metric name, since then it is easier to understand data when querying in grafana

Reply:
also, try to name similar metrics with the same prefix, that makes it easier to find with auto complete: `monitor_staking_stake_avg_gwei` and `monitor_staking_stake_median_gwei`